hudi-common/src/main/java/org/apache/hudi/common/fs/FSUtils.java

```diff
@@ -96,20 +90,6 @@ public class FSUtils {
 
   private static final StoragePathFilter ALLOW_ALL_FILTER = file -> true;
 
-  public static Configuration buildInlineConf(Configuration conf) {
```

Comment:
If it helps, can you separate the changes of moving util methods to a different set of PRs so they are easier to review?

Reply:
Discussed offline.  This change is coupled with the PR.